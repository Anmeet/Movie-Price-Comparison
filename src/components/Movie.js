import React from 'react';
import { Link } from 'react-router-dom';

const Movie = ({ id, poster, title }) => {
  return (
    <div className='movie'>
      <img src={poster} alt='' />
      <Link to={`/movie/${id}`}><span>{title}</span></Link>
    </div>
  );
};

export default Movie;
