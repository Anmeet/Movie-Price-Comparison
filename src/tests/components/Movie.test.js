import React from 'react';
import Movie from '../../components/Movie';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';

test('should render Movie Page correctly', () => {
  const wrapper = shallow(
    <Movie id='123' poster='https://poster.com' title='poster name' />
  );
  expect(toJSON(wrapper)).toMatchSnapshot();
});
