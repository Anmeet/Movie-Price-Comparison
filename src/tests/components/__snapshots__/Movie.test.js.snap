// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Movie Page correctly 1`] = `
<div
  className="movie"
>
  <img
    alt=""
    src="https://poster.com"
  />
  <[object Object]
    to="/movie/123"
  >
    <span>
      poster name
    </span>
  </[object Object]>
</div>
`;
